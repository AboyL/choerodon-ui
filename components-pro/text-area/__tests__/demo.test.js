import demoTest from '../../../tests/shared/demoTest';

demoTest('text-area-pro');
