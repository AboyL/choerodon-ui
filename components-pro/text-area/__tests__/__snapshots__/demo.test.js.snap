// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components-pro/text-area/demo/autoSize.md correctly 1`] = `
<div
  class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
>
  <label>
    <textarea
      autocomplete="off"
      class="c7n-pro-textarea"
      placeholder="适应文本高度"
      rows="4"
      style="min-rows:2px;max-rows:8px;resize:vertical"
      type="text"
    />
  </label>
</div>
`;

exports[`renders ./components-pro/text-area/demo/basic.md correctly 1`] = `
<div
  class="c7n-row"
  style="margin-left:-5px;margin-right:-5px"
>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <div
      class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
    >
      <label>
        <textarea
          autocomplete="off"
          class="c7n-pro-textarea"
          placeholder="Basic usage"
          rows="4"
          style="resize:none"
          type="text"
        />
      </label>
    </div>
  </div>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <div
      class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
    >
      <label>
        <textarea
          autocomplete="off"
          class="c7n-pro-textarea"
          placeholder="readOnly"
          readonly=""
          rows="4"
          style="resize:none"
          type="text"
        />
      </label>
    </div>
  </div>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <div
      class="c7n-pro-textarea-wrapper c7n-pro-textarea-disabled c7n-pro-textarea-empty"
    >
      <label>
        <textarea
          autocomplete="off"
          class="c7n-pro-textarea"
          disabled=""
          placeholder="disabled"
          readonly=""
          rows="4"
          style="resize:none"
          type="text"
        />
      </label>
    </div>
  </div>
</div>
`;

exports[`renders ./components-pro/text-area/demo/cols-autoSize.md correctly 1`] = `
<div
  class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
>
  <label>
    <textarea
      autocomplete="off"
      class="c7n-pro-textarea"
      placeholder="设置宽高适应"
      rows="4"
      style="min-rows:2px;max-rows:8px;min-cols:1px;max-cols:2px;resize:vertical"
      type="text"
    />
  </label>
</div>
`;

exports[`renders ./components-pro/text-area/demo/controlled.md correctly 1`] = `
<div
  class="c7n-pro-textarea-wrapper"
>
  <label>
    <textarea
      autocomplete="off"
      class="c7n-pro-textarea"
      placeholder=""
      rows="4"
      style="resize:none"
      type="text"
    >
      default
    </textarea>
  </label>
</div>
`;

exports[`renders ./components-pro/text-area/demo/dataset.md correctly 1`] = `
<div
  class="c7n-pro-textarea-wrapper c7n-pro-textarea-required"
>
  <label>
    <textarea
      autocomplete="off"
      class="c7n-pro-textarea"
      name="content"
      placeholder=""
      rows="4"
      style="resize:both"
      type="text"
    >
      textarea
    </textarea>
  </label>
</div>
`;

exports[`renders ./components-pro/text-area/demo/resizable.md correctly 1`] = `
<div
  class="c7n-row"
  style="margin-left:-5px;margin-right:-5px"
>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <div
      class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
    >
      <label>
        <textarea
          autocomplete="off"
          class="c7n-pro-textarea"
          cols="40"
          placeholder="resize both"
          rows="4"
          style="resize:both"
          type="text"
        />
      </label>
    </div>
  </div>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <div
      class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
    >
      <label>
        <textarea
          autocomplete="off"
          class="c7n-pro-textarea"
          cols="40"
          placeholder="resize vertical"
          rows="4"
          style="resize:vertical"
          type="text"
        />
      </label>
    </div>
  </div>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <div
      class="c7n-pro-textarea-wrapper c7n-pro-textarea-empty"
    >
      <label>
        <textarea
          autocomplete="off"
          class="c7n-pro-textarea"
          cols="40"
          placeholder="resize horizontal"
          rows="4"
          style="resize:horizontal"
          type="text"
        />
      </label>
    </div>
  </div>
</div>
`;
